experimental/generator-dotnet-yeoman/generators/app/templates/dotnet/botName.sln

```diff
@@ -0,0 +1,25 @@
+
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30503.244
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "<%= botName %>", "<%= botName %>\<%= botName %>.csproj", "{<%= botProjectGuid %>}"
```

Comment:
I can't remember off the top of my head - can you double check that the project type ID is the same between a web app and Functions project? If they are, then we'll need to handle it.